src/ray/protobuf/node_manager.proto

```diff
@@ -52,15 +52,17 @@ message RequestWorkerLeaseRequest {
 
 message RequestWorkerLeaseReply {
   enum SchedulingFailureType {
+    // The default failure type is "not failed".
+    NOT_FAILED = 0;
```

Comment:
why do we need to add this in this PR?

Reply:
This is not technically necessary for this PR. But setting non failure case as 0 is dangerous, so I just added it. 

if you mind it, I can create another PR haha

Reply:
I think it's fine, was just trying to understand if it was necessary